python/prairielearn_test.py

```diff
@@ -304,3 +305,33 @@ def grading_function(_: str) -> tuple[bool, Optional[str]]:
     pl.grade_answer_parameterized(question_data, question_name, grading_function)
 
     assert question_data["format_errors"][question_name] == "No answer was submitted"
+
+
+@pytest.mark.parametrize(
+    "item",
+    [
+        1,
+        [1, 2, [3, 4, {"thing": 5}]],
+        [1, 2, "999999999999999999999"],
+        [{"stuff": 999999}, {"other stuff": "key"}],
+    ],
+)
+def test_all_integers_within_limits_no_exception(item: Any) -> None:
```

Comment:
Nit: can we move these tests to a `zygote_utils_test.py` file?

Reply:
Done here: https://github.com/PrairieLearn/PrairieLearn/pull/6980/commits/ffb03392d1bdf6150c1774ef643d6df62263f33c